scripts/start-react-native.sh

```diff
@@ -14,4 +14,4 @@ if [ ! -z "$METRO_PID" ]; then
   [[ $term == 'y' ]] && kill $METRO_PID
 fi
 
-yarn start
+yarn start --reset-cache
```

Comment:
could you elaborate?

Reply:
Had to reset metro cache every time to see jest or babel related changes, so I added this line, I think it helps local development also so that we don't get random cache related bugs on dev..